src/pages/ShopingCart.page.js

```diff
@@ -28,4 +32,23 @@ export class ShopingCartPage extends BaseSwagLabPage {
   async removeCartItemById(id) {
     await this.cartItems.nth(id).locator(this.removeItemSelector).click();
   }
+
+  async getItemInfoById(id){
+    const item = this.cartItems.nth(id);
+    return super.getItemInfoById(id, item)
```

Comment:
no need to provide both here. If I got your idea. If not could you explain it in more details?

Reply:
So, I re-use getItemInfoById in shopingCart.page.js
async getItemInfoById(id){
    const item = this.cartItems.nth(id);
    return super.getItemInfoById(id, item)
  }
But to use it, I need to provide a correct locator. On inventory page - inventoryItems and in shopingCart - cartItems
And as a result I set default value and in shopingCart I change it. 

I dont see other option here(

Reply:
I think you can split this method into 2 different methods.
The first one will getItemInfoByItemElement, while another will getItemElementById